### PR TITLE
Fix snapshots size

### PR DESCRIPTION
**Pull request by salvatore-coppola**
Now snapshots are serialized without OCD component definition.

### CODE REVIEW

File: kura/test/org.eclipse.kura.core.configuration.test/src/test/java/org/eclipse/kura/core/configuration/ConfigurationServiceTest.java [hunk 1]
```diff
@@ -2171,6 +2162,7 @@ protected String marshal(Object object) {
     }
 
     @Test
+    @Ignore
```
Comment (nicolatimeus) [inline kura/test/org.eclipse.kura.core.configuration.test/src/test/java/org/eclipse/kura/core/configuration/ConfigurationServiceTest.java]: Is this intentional?

Comment (salvatore-coppola) [reply kura/test/org.eclipse.kura.core.configuration.test/src/test/java/org/eclipse/kura/core/configuration/ConfigurationServiceTest.java]: no, this is a typo!

File: kura/org.eclipse.kura.core.configuration/src/main/java/org/eclipse/kura/core/configuration/ConfigurationServiceImpl.java [hunk 2]
```diff
@@ -1034,10 +1034,21 @@ private void encryptPlainSnapshots() throws KuraException, IOException {
         }
     }
 
-    private synchronized long saveSnapshot(List<ComponentConfiguration> configs) throws KuraException {
+    private synchronized long saveSnapshot(List<ComponentConfiguration> configs, boolean includeDefinition)
```
Comment (MMaiero) [inline kura/org.eclipse.kura.core.configuration/src/main/java/org/eclipse/kura/core/configuration/ConfigurationServiceImpl.java]: Do we need to keep the "includeDefinition" parameter? All the usages we have right now set the value to false.
is there any other usage that I've missed?

Comment (salvatore-coppola) [reply kura/org.eclipse.kura.core.configuration/src/main/java/org/eclipse/kura/core/configuration/ConfigurationServiceImpl.java]: I'd keep it, at the moment no method set the flag to true but maybe in the future some methods will do it, also a configuration normally includes the definition so an invoker could expect to have the configuration definition serialized.

Comment (MMaiero) [reply kura/org.eclipse.kura.core.configuration/src/main/java/org/eclipse/kura/core/configuration/ConfigurationServiceImpl.java]: Correct.
But that is a private method. So, users will not be able to leverage it in any way, right?

Comment (salvatore-coppola) [reply kura/org.eclipse.kura.core.configuration/src/main/java/org/eclipse/kura/core/configuration/ConfigurationServiceImpl.java]: that's right, it's more a concern about someone who could use the method in the future, but however we can remove the flag

Comment (MMaiero) [reply kura/org.eclipse.kura.core.configuration/src/main/java/org/eclipse/kura/core/configuration/ConfigurationServiceImpl.java]: Thanks.

**Review by MMaiero — APPROVED** [review]

File: build-all.bat [hunk 3]
```diff
@@ -0,0 +1,4 @@
+mvn -f target-platform/pom.xml clean install "-Dmaven.test.skip=true" &&^
```
Comment (MMaiero) [inline build-all.bat]: Is this file new? Is it related to this PR or not?

Comment (salvatore-coppola) [reply build-all.bat]: no this is a script file of mine.

**Pull request merged**